samples/java-webmvc/build.gradle

```diff
@@ -1,13 +1,15 @@
 buildscript {
     ext {
+        springAutoRestDocsVersion = "2.0.1"
```

Comment:
I guess we want to base examples on snapshot version and bump with each release just like everything else.
The main project documentation should contain link to repo at the latest stable release tag, which will contain example projects based on latest release version. but during development, we need to utilize snapshots everywhere.

Reply:
I adjusted all example project build to use the snapshot version. In addition, I upgraded them to Spring Boot 2.0.1.